mindsdb/integrations/handlers/couchbase_handler/couchbase_handler.py

```diff
@@ -195,6 +195,7 @@ def get_tables(self) -> Response:
         for _scope in bucket.collections().get_all_scopes():
                 for __collections in _scope.collections:
                     collections.append(__collections.name)
+        collections = list(set(collections))
```

Comment:
Maybe we can simplify this by using set in the beginning and then convert it to list, instead of list => set => list. e.g

```
unique_collections = set()

for scope in bucket.collections().get_all_scopes():
    for collection in scope.collections:
        unique_collections.add(collection.name)

collections = list(unique_collections)
```

Reply:
I fixed that. You can check it here 27b5ff2b480cc8a42b685f9acf8097198cbcc0c8 . Please let me know if you need something else.